app/views/sections/_form.html.erb

```diff
@@ -8,6 +8,11 @@
     <%= f.label :week, 'Section week' %>
     <%= f.text_field :week, class: 'form-control' %>
   </div>
+  <div class="form-group">
+    <%= f.label :github_path, 'Section Github URL' %>
```

Comment:
"github_path" ties this to Github. If we ever changed hosting services, we'd have to change the name Github everywhere to avoid confusion. Maybe come up with a more generic name instead?

Reply:
added to project to do list on trello